style-guide/styles/wds-mixins/_button-theming.scss

```diff
@@ -15,6 +15,16 @@
 		$color-secondary-active: darken($color-secondary, 20%);
 	}
 
+	@if ($manual-override) == "lighter" {
```

Comment:
can we add these conditions to `if` statements above and use `or`?

Reply:
@JamesHawking what about @latata's comment?